modules/nf-core/plastid/metagene_generate/main.nf

```diff
@@ -0,0 +1,44 @@
+process PLASTID_METAGENE_GENERATE {
+    tag "$gtf"
+    label "process_low"
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plastid:0.6.1--py39had3e4b6_2':
+        'biocontainers/plastid:0.6.1--py39had3e4b6_2' }"
+
+    input:
+    path gtf
```

Comment:
Both input channels will need a meta, you probably just want one tuple channel with both